packages/monorepo-scripts/src/constants.ts

```diff
@@ -5,5 +5,7 @@ export const constants = {
     stagingWebsite: 'http://staging-0xproject.s3-website-us-east-1.amazonaws.com',
     lernaExecutable: path.join('node_modules', '@0x-lerna-fork', 'lerna', 'cli.js'),
     githubPersonalAccessToken: process.env.GITHUB_PERSONAL_ACCESS_TOKEN_0X_JS,
+    discordAlertWebhookUrl: process.env.DISCORD_GITHUB_RELEASE_WEBHOOK_URL,
```

Comment:
Does this need to be an ENV var? Why not just hard-code it here? Don't think it will be changing frequently.

Reply:
This will be a URL that will allow the posting to the #announcements ( or other channel ) under the name of something like "0x Release Bot".  We shouldn't expose this URL publicly, because malicious folks could use it to spam.